src/ds5/ds5-options.cpp

```diff
@@ -732,12 +734,62 @@ namespace librealsense
         if (res.empty())
             throw invalid_value_exception("auto_exposure_limit_option::query result is empty!");
 
-        return *(reinterpret_cast<uint32_t*>(res.data() + 4));
+        return static_cast<float>(*(reinterpret_cast<uint32_t*>(res.data() + 4)));
     }
 
     option_range auto_gain_limit_option::get_range() const
     {
         return *_range;
     }
 
+    librealsense::thermal_compensation::thermal_compensation(
+        std::shared_ptr<ds5_thermal_monitor> monitor,
+        std::shared_ptr<option> toggle) :
+        _thermal_monitor(monitor),
+        _thermal_toggle(toggle)
+    {
+        try {
```

Comment:
formatting

Reply:
No need here  - part of ctor

Reply:
But we do have coding conventions stating `{` on its own line, no?